cmd/kube-scheduler/app/server.go

```diff
@@ -234,7 +234,6 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}, regis
 	}
 
 	// Start all informers.
-	go cc.PodInformer.Informer().Run(stopCh)
```

Comment:
If you remove this and rely on cc.InformerFactory.Start(), you will get all pods update.